samples/FunctionApp/Program.cs

```diff
@@ -21,9 +23,17 @@ static async Task Main(string[] args)
                 .ConfigureFunctionsWorkerDefaults(builder =>
                 {
                     builder
-                        .AddApplicationInsights()
+                        .AddApplicationInsights(options =>
+                        {
+                            // Configure the underlying ApplicationInsightsServiceOptions
+                            options.EnableAdaptiveSampling = false;
```

Comment:
Is this something we want to recommend to customers? It can cause *a lot* of telemetry to be sent up. Depending on what their app does, they may quickly hit their daily cap without it.

In my experience when I disabled it for an aspnetcore app, I hit 100GB in only a few hours. We should at least add a comment explaining the considerations and tradeoffs the customer should make before disabling this.

Reply:
Changed the property to a less significant one. My intent is just to make it obvious and show how to configure options.

Reply:
If that is the case, can you add a comment making that explicit?

``` CSHarp
// The below property configuration is purely for demonstration purposes. It is not required you set the same property to the same value in your application.
```

Reply:
Added comment.